### PR TITLE
output tree contents to outputPS4.txt

### PR DESCRIPTION
**Pull request by shu1997**

### CODE REVIEW

File: src/com/company/Main.java [hunk 1]
```diff
@@ -115,12 +115,13 @@ public static void main(String[] args) {
...
         System.out.println();
         sortedUsers.forEach((book) -> book.print());
+        printToOutputFile(sortedUsers); //print tree nodes content sorted in ASC order of Book Id to outputPS4.txt
     }
 
     public static void _readBookList() {
         BufferedReader reader;
         try {
-            reader = new BufferedReader(new FileReader("inputsPS4.txt")); // providing the name of file from the same directory
+            reader = new BufferedReader(new FileReader("inputPS4.txt")); // providing the name of file from the same directory
```
Comment (22shubham22) [inline src/com/company/Main.java]: inputsPS4.txt hi hoga. pdf me vhi diya hua h

File: src/com/company/Main.java [hunk 2]
```diff
@@ -133,6 +134,21 @@ public static void _readBookList() {
         }
     }
 
+    public static void printToOutputFile(List<BookNode> list){ // printing list content to outputPS4.txt
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter("outputPS4.txt"));
+            for(int i=(list.size()-1);i>=0;i--) {
+                // printing data of each book
+                writer.write("Book Id: "+list.get(i).bookId+", AvailableCounter: "+list.get(i).availableCount+", CheckoutCounter: "+list.get(i).checkoutCounter+"\n");
+            }
```
Comment (22shubham22) [inline src/com/company/Main.java]: ulta kyu print kr rhe ho ?

Comment (shu1997) [reply src/com/company/Main.java]: ascending order me krna hai sort hua hai desc mein

File: src/com/company/Main.java (hunk 1 above)
Comment (shu1997) [reply src/com/company/Main.java]: lekin file ka naam hai inputPS4.txt

Comment (shu1997) [reply src/com/company/Main.java]: project structure me

Comment (22shubham22) [reply src/com/company/Main.java]: to file ka naam inputsPS4.txt kr k usko v uplaod kr do

File: src/com/company/Main.java (hunk 2 above)
Comment (22shubham22) [reply src/com/company/Main.java]: nhi nhi hm kisi aur parameter pr sort kiye h bookid se nhi. tm normally print kro

**Pull request merged**